hyperstart.go

```diff
@@ -315,6 +347,38 @@ func buildHyperContainerProcess(cmd Cmd) (hyperJson.Process, error) {
 	return process, nil
 }
 
+func isStarted(c net.Conn, chType chType) bool {
```

Comment:
We probably only need to check for one channel only. I can't see a case where we'd have only one channel alive and not the other.